webservices/WFLY-13356_EAP7-1525.adoc

```diff
@@ -0,0 +1,105 @@
+= WFLY-13356/ EAP7-1525 Add WebServices Galloen layer
+:author:            Jim Ma
+:email:             ema@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+The goal of this feature is to supply a https://docs.wildfly.org/galleon/#_layers[Galleon layer] which will be responsible for providing the webservices subsystem.
+
+The following is a configuration example of the WildFly bootable JAR Maven plugin to provision a server with the `webserviecs` layer and the `cloud-server` layer, which brings the webseviecs functionatily to support Jakarta xml web service deployment.
+
+[source,xml]
+----
+<plugin>
+    <groupId>org.wildfly.plugins</groupId>
+    <artifactId>wildfly-jar-maven-plugin</artifactId>
+    <configuration>
+        <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#${version.wildfly}</feature-pack-location>
+        <layers>
+            <layer>cloud-server</layer>
+            <layer>webservices</layer>
+        </layers>
+    </configuration>
+    <executions>
+        <execution>
+            <goals>
+                <goal>package</goal>
+            </goals>
+        </execution>
+    </executions>
+</plugin>
+----
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-1525[EAP7-1525]
+* {issue-base-url}/WFLY-13356[WFLY-13356]
+
+=== Related Issues:
+
+* {issue-base-url}/WFLY-13354[WFLY-13354]
+* {issue-base-url}/WFLY-13771[WFLY-13771]
+
+=== Dev Contacts:
+
+* mailto:ema@redhat.com[Jim Ma]
+
+=== QE Contacts:
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+* mailto:tborgato@redhat.com[Tommaso Borgato]
+
+=== Testing By
+
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components:
+
+* WildFly
+
+== Requirements
+
+WildFly Galleon is the tool to slim the WidlFly server installation. To better support this tool and provide user with wide variety of server provisioning selection, webservices layer should be created to provide Jakarta xml web service support.
+The *webserviecs* galloen layer spec should be created, and it should depend on *ejb3* layer to support EJB(type) WebSevice and *messaging-activemq* layer to support jms transport.
```

Comment:
@jimma is that a hard requirement? there is no such use case as a user not using JMS for example? are these layers optional to support such a use case?

Reply:
If user's webservice deployment is pojo webservice, these two ejb and jms are really not needed.

Reply:
and is the user able to provide a slimmed server without them?

Reply:
Do you mean exclude "ejb" and "jms" layer ?

Reply:
yes

Reply:
I guess this answers my questions: 
```<layer name="web-server"/>
        <layer name="ejb-lite" optional="true"/>
        <layer name="messaging-activemq" optional="true"/>```